### ISSUE TITLE
JS Custom is disabling sublimelinter-eslint

### ISSUE THREAD

**Issue by ellipticaldoor** [Thom1729/Sublime-JS-Custom#23]
This is my syntax_map in sublimelinter config

``` json
	"syntax_map": {
		"html (django)": "html",
		"html (rails)": "html",
		"html 5": "html",
		"javascript (babel)": "javascript",
		"magicpython": "python",
		"php": "html",
		"python django": "python",
		"pythonimproved": "python",
		"naomi.fjsx15": "javascript",
		"naomi.html5": "html",
		"naomi.mql4": "mql",
		"naomi.php7": "php",
		"naomi.scss3": "scss",
		"javascript (babel)": "javascript",
		"javascript (dom)": "javascript"
	},
```

I need to add JS Custom to this list but I don't know which name I should use it. Do you you how I can make it work?

**Comment by ellipticaldoor** [comment]
Yes, I'm using latest sublimelinter

**Comment by Thom1729** [comment]
This is an issue with SublimeLinter-eslint. [The fix](https://github.com/SublimeLinter/SublimeLinter-eslint/commit/c40376c36312e582704b4fafbc36f4b17171394f#diff-6c1f4f1574c85da37dc4dd59341c9a39) was merged yesterday, but it's not in a release yet. You can patch it yourself by applying the following to your `SublimeLinter.sublime-settings`:

```json
// SublimeLinter Settings - User
{
    "linters": {
        "eslint": {
            "selector": "source.js"
        }
    },
}
```

**Comment by Thom1729** [comment]
Fixed in [SublimeLinter-eslint 4.0.3](https://github.com/SublimeLinter/SublimeLinter-eslint/releases/tag/4.0.3).
